source/_includes/asides/docs_navigation.html

```diff
@@ -124,11 +124,12 @@ <h1 class="title delta">Topics</h1>
       <li>
         <b>{% active_link /docs/frontend/ Frontend %}</b>
         <ul>
-          <li>
-            {% active_link /docs/frontend/browsers/ Browser compatibility list %}
-          </li>
           <li>{% active_link /dashboards Dashboards %}</li>
+          <li>{% active_link /docs/organizing/ Organizing %}</li>
           <li>{% active_link /docs/frontend/icons/ Icons%}</li>
+          <li>
+            {% active_link /docs/frontend/browsers/ Browser compatibility list%}
+          </li>
```

Comment:
Not for this PR, just a note: But I think we should consider ditching this (if we want a replacement, a generic "system requirements page" might be a better fit).